CrewChiefV4/Events/LapCounter.cs

```diff
@@ -206,42 +204,30 @@ private void playPreLightsMessage(GameStateData currentGameState, int maxNumberT
                         currentGameState.PitData.PitWindowStart), 0, this));
                 }
             }
-            if (CrewChief.gameDefinition.gameEnum != GameEnum.RF2_64BIT)
+            if (currentGameState.SessionData.ClassPosition == 1)
             {
-                if (currentGameState.SessionData.ClassPosition == 1)
+                Console.WriteLine("Pre-start message for pole");
+                if (SoundCache.availableSounds.Contains(Position.folderDriverPositionIntro))
                 {
-                    Console.WriteLine("Pre-start message for pole");
-                    if (SoundCache.availableSounds.Contains(Position.folderDriverPositionIntro))
```

Comment:
this if / else block is an old hack to work around some issue in Gianfranco's sound pack. I think needs to be kept